### DISCUSSION TITLE
Table help

### DISCUSSION THREAD

**Discussion by MelB507** [community/community#46697] [New to GitHub]
Hi, i need some help if possible with tables in GitHub docs.

1/ I have a table with 4 columns and they do not all have the same amount of text.  The cells with the least amount of text is aligning to the middle even when i have aligned it to the left (blank gaps at the top).  How can i remove the gaps at the top and align top left?

2/ How do i add a single row along the bottom of a table that has 4 columns?

3/ How can i extend the length of a table row without adding a full row of text?

4/ Why is there no toolbar for this basis stuff?

**Comment by Juanescacha** [comment]
Can you share the code and also the images of the tables?

**Comment by MelB507** [comment]
The text seems to wrap and will not align to the top left of the column.Also, i wanted to add a row at the bottom of the four colums that is the same length as the four cols above   This is how I want it to look    I appreciate any help you can offer.  Warmest regards Mel    From: Juan Esteban CamargoSent: Saturday, 11 February 2023 8:29 AMTo: community/communityCc: MelB507; ManualSubject: Re: [community/community] Table help (Discussion #46697) Can you share the code and also the images of the tables?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

**Comment by Juanescacha** [comment]
okey but this is css code for html. not code for github tables, can you send html?
